.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+Metrics/BlockLength:
+  Enabled: false
+
+Lint/BinaryOperatorWithIdenticalOperands:
+  Enabled: false
```

Comment:
- [ ] 🛑 don't edit rubocop configs, please get it back to what's provided by Microverse [here](https://github.com/microverseinc/linters-config/tree/master/ruby)
_we use a tool to check this, so please don't try to sneak a line in there_

Reply:
I didn't try to sneak anything. We just didn't know microverse provided a config file.

Reply:
@elberthcorniell sorry if that came out wrong, I didn't accuse you of that, I'm just letting you know